src/Tests/Microsoft.Diagnostics.Monitoring.TestCommon/TestAppScenarios.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Microsoft.Diagnostics.Monitoring.TestCommon
+{
+    public static class TestAppScenarios
+    {
+        public static class Commands
+        {
+            public const string EndScenario = nameof(EndScenario);
+            public const string StartScenario = nameof(StartScenario);
+        }
+
+        public static class SpinWait
```

Comment:
Do you we have to use the spin scenario for our test apps? It seems incredibly cpu intensive, but i'm not sure what benefit it provides for tests such as /processes.

Reply:
I can create a non spinning wait, more of an async wait. We'll probably want a spin wait for testing high cpu usage in metrics and eventually triggers. I'll change this to an async wait scenario.

Reply:
Changed to async wait.